ui/src/components/UHGroupingsInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import AboutInfoItem from '@/components/AboutInfoItem';
+
+const UHGroupingsInfo = ({h1Color, h1Weight, AboutInfoItemSize}: { h1Color: string, h1Weight: string, AboutInfoItemSize?: string }) => {
```

Comment:
I think we should go with one prop instead of three. A `variant` prop where there are two possibilities: `default` or `large`.

Reply:
done